src/modules/home/TestimonialsBanner.tsx

```diff
@@ -153,14 +151,25 @@ function TestimonialsBanner() {
                     </Text>
                   </div>
                 </Flex>
-                <Text
-                  className={composeClasses(
-                    'text-xs sm:text-base',
-                    extendedPalette.componentText
-                  )}
-                >
-                  {testimonial.message}
-                </Text>
+                <div>
+                  <div
+                    className={`${
```

Comment:
Should be more readeable with composeClasses here

```suggestion
                    className={`${
```